### PR TITLE
Update uwp.md

### PR DESCRIPTION
**Pull request by andrew-mi**

### CODE REVIEW

**Comment by PRMerger20** [comment]
@andrew-mi : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by andrew-mi** [comment]
I'm sure this could be better worded to keep the docs clear. But I do think this is useful information to include here.

File: docs/sdk/troubleshooting/uwp.md [hunk 1]
```diff
@@ -34,9 +34,9 @@ ms.tgt_pltfrm: uwp
    AppCenter.LogLevel = LogLevel.Verbose;
    ```
 
-5. Make sure your device is online.
-6. At times, logs might take few minutes to surface in the portal. Wait for some time if that’s the case.
-7. To check if App Center backend received your data, go to the Log flow section in Analytics service. Your events should appear once it's been sent.
+5. Make sure your device is online and see whether you can access https://in.appcenter.ms/ in a browser. It should load as a **blank page**. If instead you get a connection error, you should check whether that domain has been blocked/filtered on your network.
+7. At times, logs might take few minutes to surface in the portal. Wait for some time if that’s the case.
```
Comment (lucen-ms) [inline docs/sdk/troubleshooting/uwp.md]: Please fix numbering.

**Comment by PRMerger10** [comment]
@andrew-mi : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by lucen-ms — APPROVED** [review]

**Comment by lucen-ms** [comment]
Thank you @andrew-mi for enhancing our documentation!

**Pull request merged**